src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/SqlCommandIntegration.cs

```diff
@@ -64,7 +84,7 @@ public static class SqlCommandIntegration
                     moduleVersionPointer: moduleVersionPtr,
                     mdToken: mdToken,
                     opCode: opCode,
-                    instrumentedType: SqlCommandTypeName,
+                    instrumentedType: SystemSqlCommandTypeName,
```

Comment:
And here, this could be `MicrosoftSqlCommandTypeName` right?

Reply:
Yup! Fixed in 8e258e04.